src/util/sfdc.ts

```diff
@@ -120,4 +120,21 @@ export const sfdc = {
   matchesAccessToken: (value: string): boolean => {
     return /^(00D\w{12,15})![.\w]*$/.test(value);
   },
+
+  /**
+   * Tests whether a given url is an internal Salesforce domain
+   *
+   * @param url
+   */
+  isInternalUrl: (url: string): boolean => {
+    const INTERNAL_URL_PARTS = [
+      '.internal.',
+      '.vpod.',
+      'stm.salesforce.com',
+      '.blitz.salesforce.com',
```

Comment:
I don't think these test environments are available anymore; blitz* and mobile*. Blitz* was replaced with STEAM stmf* envs and mobile* with STREAM stmp* envs.

Reply:
should we update them?  Have you got a list like the others you sent me for falcon domains?

Reply:
I searched configs and blitz is still present, but mobile is not. I say remove mobile, then QA.